tests/files/vagrant_ubuntu18-calico-dual-stack.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Kubespray settings
+
+kube_network_plugin: calico
+enable_dual_stack_networks: true
+
+deploy_netchecker: true
```

Comment:
Is netchecker checking dualstack ?

Reply:
I'm not familiar with netchecker. I can drop if you don't think it needs to be there?

Reply:
I want to keep it, just wondering if it check dual stack, or if there is a setting to make it check, or if maybe we need an IPv4 and an IPv6 deployment. Maybe someone else knows, this is an open question not a blocker to merge.